packages/components/src/Tooltip/tests/chromatic/Tooltip.stories.tsx

```diff
@@ -0,0 +1,203 @@
+            </TooltipTrigger>
+            <TooltipTrigger defaultOpen placement="end">
+                <Button>{buttonText}</Button>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+            <TooltipTrigger defaultOpen placement="right">
+                <Button>{buttonText}</Button>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+            <TooltipTrigger defaultOpen placement="left">
+                <Button>{buttonText}</Button>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+            <TooltipTrigger defaultOpen placement="top">
+                <Button>{buttonText}</Button>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+            <TooltipTrigger defaultOpen placement="bottom">
+                <Button>{buttonText}</Button>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+        </Grid>
+    )
+} satisfies Story;
+
+export const ShouldFlip = {
+    render: args => (
+        <Stack>
+            <H1>Original Placement: left</H1>
+            <TooltipTrigger defaultOpen placement="left">
+                <Button>{buttonText}</Button>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+        </Stack>
+    ),
+    decorators: [
+        Story => (
+            <Flex justifyContent="left">
+                <Story />
+            </Flex>
+        )
+    ],
+    parameters: {
+        skipGlobalDecorator: true
+    }
+} satisfies Story;
+
+export const LinkTrigger = {
+    render: args => (
+        <TooltipTrigger defaultOpen>
+            <Link>{buttonText}</Link>
+            <Tooltip {...args} />
+        </TooltipTrigger>
+    )
+} satisfies Story;
+
+export const AvatarTrigger = {
+    render: function Render(args) {
+        return (
+            <TooltipTrigger defaultOpen>
+                <PassiveTrigger>
+                    <Avatar name="Fred Allen" />
+                </PassiveTrigger>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+        );
+    }
+} satisfies Story;
+
+export const LongContent = {
+    render: args => (
+        <TooltipTrigger isOpen>
+            <Button>{buttonText}</Button>
+            <Tooltip {...args} />
+        </TooltipTrigger>
+    ), 
+    args: {
+        children: `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam in turpis
+                        ac libero tincidunt hendrerit. Ut vitae nisl nec orci laoreet tristique.
+                        Nulla facilisi. Nulla facilisi. Nulla facilisi. Nulla facilisi. Nulla
+                        facilisi. Nulla facilisi. Nulla facilisi. Nulla facilisi. Nulla facilisi.`
+    },
+    decorators: [
+        Story => (
+            <Flex UNSAFE_marginTop="8rem" justifyContent="center">
+                <Story />
+            </Flex>
+        )
+    ],
+    parameters: {
+        skipGlobalDecorator: true
+    }
+} satisfies Story;
+
+export const DisabledTrigger = {
```

Comment:
Is the PassiveTrigger approach working with disabled listbox items? in a select for instance. I know it's a use case i've seen

Reply:
I haven't tested but as long as it's inside the list item, it should work. I think... i can do a test if you'd like.